### DISCUSSION TITLE
Just found Umami. WOW! But can I use on Wordpress?!

### DISCUSSION THREAD

**Discussion by terry9873** [umami-software/umami#1705] [Q&A]
I am desperate for some basic website analytics, but all the guides seem to be for server admins and coders. I am neither!

I run various WP websites with WP shared or managed hosting accounts. 

Is it possible to get Umami running without code/server knowledge? 
thanks

**Comment by briancao** [comment]
Umami requires at least some code/server knowledge to setup, and more to troubleshoot.  

You can try following the guides (https://umami.is/docs/hosting) to host on a provider like Railway, which has a UI forward model to spin up the App and DB.

Otherwise I suggest just signing up for Umami Cloud, which is currently in open beta.
